Cargo.toml

```diff
@@ -23,10 +23,18 @@ keywords = ["linear-algebra"]
 changelog = "CHANGELOG.md"
 
 [features]
+# apple accelerate
 accelerate = ["accelerate-src"]
-intel-mkl = ["intel-mkl-src"]
+# parallel LP64 version of MKL
+intel-mkl = ["intel-mkl-src/mkl-static-lp64-iomp"]
+# sequential LP64 version of MKL
+intel-mkl-seq = ["intel-mkl-src/mkl-static-lp64-seq"]
```

Comment:
I think we agreed to have one and leave the rest to the user.

Reply:
But then do you still think that the default should be the parallel one and not the sequential?

Reply:
Since `intel-mkl-src` itself defaults to parallel, we might as well do that, too:

https://github.com/rust-math/intel-mkl-src/blob/master/intel-mkl-src/build.rs#L54

We are then simply adjusting it to be lp64 instead of ilp64.